crds/src/lib.rs

```diff
@@ -28,6 +32,24 @@ pub struct ConfidentialClusterSpec {
     pub register_server_port: Option<i32>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
+pub struct ConfidentialClusterStatus {
+    pub conditions: Vec<Condition>,
+}
+
+pub const INSTALLED_CONDITION: &str = "Installed";
+pub const KNOWN_TRUSTEE_ADDRESS_CONDITION: &str = "KnownTrusteeAddress";
+pub const ADDRESS_FOUND_REASON: &str = "AddressFound";
+pub const NO_ADDRESS_FOUND_REASON: &str = "NoAddressFound";
```

Comment:
Can be those be merged into one?

Reply:
Whoops, had removed the uses but not the definition. Fixed now.